README.md

```diff
@@ -2,20 +2,21 @@
 
 You have a process which is done manually or even on paper and you want to move into the digital world.
 
-Your process has the following characteristics:
+Your process has...
 
-* Centered around a form (e.g. application, certification, ...)
-* Different users involved
-* Specific flow how a form is processed
+* a form based focus (e.g. application, certification, ...)
```

Comment:
hmm... based and focus is kind of the same... maybe we should think of something else.